prisma/schema.prisma

```diff
@@ -24,15 +24,25 @@ model Recipe {
   id Int @default(autoincrement()) @id
   name String
   description String
-  imageKey String? @map("image_key")
   author User @relation(fields: [authorId], references: [id])
   authorId Int @map("author_id")
   preparing RecipePreparationSteps[]
   ingredients RecipeIngredients[]
+  file File? @relation(fields: [fileId], references: [id])
+  fileId Int? @unique @map("file_id")
 
   @@map("recipe")
 }
 
+model File {
+  id Int @default(autoincrement()) @id
+  name String
+  key String
+  recipe Recipe?
```

Comment:
I don't think referencing the recipe here is necessary. See prisma documentation. It's possible to define one to one relation on one side (in this case it should be on recipe side)
https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-one-relations